.github/workflows/eluc.yml

```diff
@@ -0,0 +1,35 @@
+# This runs the unit tests for the ELUC use case
+
+name: ELUC Use Case Python Unit Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./use_cases/eluc
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.10"
+    - name: Set PYTHONPATH
+      run: echo "PYTHONPATH=$PWD" >> $GITHUB_ENV
+    - name: Test PYTHONPATH
+      run: printenv PYTHONPATH
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Run unit tests
+      run: python -m unittest
+    - name: Lint with PyLint
```

Comment:
You should run `pylint` before the unit tests. First check the code is 'clean' (fast), then run the tests (slow).

Reply:
Moved